artemis-core/artemis/src/main/java/com/artemis/AspectSubscriptionManager.java

```diff
@@ -57,12 +79,42 @@ protected void setWorld(World world) {
      * See https://github.com/junkdog/artemis-odb/issues/551
 	 *
 	 * @param builder Aspect to match.
+	 * @param applyDefaults whether to apply default aspect
 	 * @return {@link EntitySubscription} for aspect.
 	 */
-	public EntitySubscription get(Aspect.Builder builder) {
+	EntitySubscription get(Aspect.Builder builder, boolean applyDefaults) {
+		if (applyDefaults && defaultAspect != null && builder.defaults) {
+			builder = applyDefaultAspect(builder.copy());
+		}
+		
 		EntitySubscription subscription = subscriptionMap.get(builder);
 		return (subscription != null) ? subscription : createSubscription(builder);
 	}
+	
+	private Aspect.Builder applyDefaultAspect(Aspect.Builder builder) {
```

Comment:
I also cannot think of a decent use case for a default oneType.

In general I think the role (all, one, exclude) of any default component should be applied unless the component is defined by the actual aspect in any of those three roles. But I'm only saying that because I already implemented it that way. Seems the most reasonable implementation to me.

Reply:
Seems like the cleanest solution, it just needs to be documented.